### PR TITLE
Tbolt/4511 Private Contractor Example Help Text Update

### LINKED ISSUE #4511: [Feature] Help Text Changes for Private Contractor Costs
### Description and related issues
Based on design ticket #4381 

### Mockups or link to Figma
https://www.figma.com/file/hJpKHKU6fz5J0Z7fisSwa2/eAPD-MMIS-2022?node-id=3328%3A18573&t=VXupikAaT7lnGE7V-1

Private Contractor Costs

### Acceptance criteria

- [x] change example box to
>  **Example:** Contractor A, Total Contract Costs $2,000,000, Full Contract Term: 4/15/2021 - 04/14/2025; FFY 2023 Cost: $100,000, FFY 2024 Cost: $300,000.

### PR DESCRIPTION
**Pull request by tbolt**
Resolves #4511 

### Description
Update the example help text for Private Contractor in MMIS APDs

### Steps to manually verify this change

1. Open MMIS APD
2. Verify example help text is updated when adding a new private contractor according to ticket
3. Verify HITECH APD example help text in private contractor costs remains the same

### This pull request is ready to code review when

- [x] Automated tests are updated (and all tests are passing)
- [x] Pull request has been labeled, if applicable with feature, content, bug,
      tests, refactor
- [x] The experience passes a basic manual accessibility audit (keyboard nav,
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when

- [ ] Code has been reviewed by someone other than the original author

### This pull request is ready to review when the QA has

- [ ] Verified the functionality related to the change
- [ ] Verified that the change works with Narrator on Windows
- [ ] Verified that the change works with VoiceOver on Mac
- [ ] Verified all updated pages with the WAVE tool
- [ ] Verified tab and keyboard navigation functionality

### This pull request can be merged when

- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

File: web/src/pages/apd/activities/contractor-costs/ContractorResources.test.js [hunk 1]
```diff
@@ -87,6 +90,7 @@ describe('the ContractorResources component', () => {
         {
           apd: {
             data: {
+              apdType: 'HITECH',
```
Comment (thetif) [inline web/src/pages/apd/activities/contractor-costs/ContractorResources.test.js:93]: use the APD_TYPE.HITECH const here

File: web/src/pages/apd/activities/contractor-costs/ContractorResources.test.js [hunk 2]
```diff
@@ -100,7 +104,8 @@ describe('the ContractorResources component', () => {
       )
     ).toEqual({
       list: [{ key: '123' }],
-      years: ['2022', '2023']
+      years: ['2022', '2023'],
+      apdType: 'HITECH'
```
Comment (thetif) [inline web/src/pages/apd/activities/contractor-costs/ContractorResources.test.js:108]: APD_TYPE.HITECH const

**Review by thetif — APPROVED** [review]

**Review by amyd11 — APPROVED** [review]

**Review by mirano-darren — APPROVED** [review]
Looks good to me!
| Given  | When | Then | Result |
| ------------- | ------------- | ------------- | ------------- |
| A MMIS APD  | Private Contractor page  | Help text is updated | ✅ |
| A HITECH APD  | Private Contractor page  | Help text is the same as before| ✅ |

**Review by stephanieboydcms — APPROVED** [review]
Good to move forward, thanks @Ty!

**Pull request merged**